source/admin_manual/debugging/debugging_rawlog.rst

```diff
@@ -33,7 +33,7 @@ Pre-login rawlog
 .. versionadded:: v2.3.2
 
 The pre-login rawlog is used before IMAP, POP3, Submission or ManageSieve
-client logs in to the post-login process. Note that LMTP and doveadm protocols
+client logs into the post-login process. Note that LMTP and doveadm protocols
```

Comment:
As noted, `in to` / `into` has a high false-positive rate in general.

In this case, I'm fairly certain that

`The ... rawlog is used before a <client logs in> <to the something>` does not do what is intended, and the proper preposition really is:
`The ... rawlog is used before a <client logs into> <the something>`